lib/modules/dosomething/dosomething_reportback/dosomething_reportback.views_default.inc

```diff
@@ -130,18 +130,21 @@ function dosomething_reportback_views_default_views() {
   $handler->display->display_options['fields']['status']['id'] = 'status';
   $handler->display->display_options['fields']['status']['table'] = 'dosomething_reportback_file';
   $handler->display->display_options['fields']['status']['field'] = 'status';
-  $handler->display->display_options['fields']['status']['label'] = 'Item Status';
-  /* Field: Reportback: Flagged status */
-  $handler->display->display_options['fields']['flagged']['id'] = 'flagged';
-  $handler->display->display_options['fields']['flagged']['table'] = 'dosomething_reportback';
-  $handler->display->display_options['fields']['flagged']['field'] = 'flagged';
-  $handler->display->display_options['fields']['flagged']['relationship'] = 'rbid';
-  $handler->display->display_options['fields']['flagged']['label'] = 'Reportback flagged status';
```

Comment:
gotcha -- ill just make it again cause its quick but not sure how to remove the feature first?